### PR TITLE
Release - Prepping 1.2.0

### PR DESCRIPTION
**Pull request by cnoon**
This PR makes all the final changes to prep for the 1.2.0 release of the Swift 2.3 branch. Once this is merged into the `swift2.3` branch, I'll then cut the official release by moving it into `master`.

### CODE REVIEW

File: Example/iOS Example/WillowConfiguration.swift [hunk 1]
```diff
@@ -35,14 +35,16 @@ struct WillowConfiguration {
     // MARK: - Formatters
 
     private struct PrefixFormatter: Formatter {
-        let prefix: String
+        let emoticons: String
+        let name: String
 
-        init(prefix: String) {
-            self.prefix = prefix
+        init(emoticons: String, name: String) {
```
Comment (EricAppel) [inline Example/iOS Example/WillowConfiguration.swift]: emoji would be a more accurate term here rather than emoticon. I looked it up to verify 😀

Comment (cnoon) [reply Example/iOS Example/WillowConfiguration.swift]: On it.

Comment (cnoon) [reply Example/iOS Example/WillowConfiguration.swift]: Addressed in 8fdcdc56.

**Comment by cnoon** [comment]
All comments have been addressed. I'm going to move this through guys. Thanks! 🍻

**Pull request merged**